### DISCUSSION TITLE
How to mention 'responeType' as 'blob' in feathersjs to download and open the file with blob object ?

### DISCUSSION THREAD

**Discussion by pankajfin** [feathersjs/feathers#2294] [Q&A]
If below format is used in axios calls , how can we achieve the same via feathersjs api call ? 
Where do we need to specify the 'responseType' key ?

` return axios({
    method: 'get',
    url: '_URL_',
    responseType: blob
  });
}`
 
I tried mentioning below code block for post request via create method from feathersjs doc:

` var config = 
      {
       headers: { 'Content-Type': 'application/json'},
       responseType: 'blob'
      }`
 `const response = _SERVICE_NAME_.create(data, config)`

Please help me to understand the format for mentioning responseType in request in feathersjs.

**Comment by wz5899** [comment]
Use
`var config = { headers: { 'Content-Type': 'application/json' }, connection: { responseType: 'blob' } }`
